enhancements/ingress/set-delete-http-headers.md

```diff
@@ -732,32 +732,51 @@ We don't allow setting the `strict-transport-security` header on either an Ingre
+
+| Simplified combination | When this comb. is present                                                                                                                                                  | This value is used                                                                                                               |
+|------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|
+| ABCD                   | haproxy.router.openshift.io/set-forwarded-headers, spec.httpHeaders.forwardedHeaderPolicy, spec.httpHeaders.actions in IngressController, spec.httpHeaders.actions in Route | spec.httpHeaders.actions in Route.                                                                                               |
+| ABC                    | haproxy.router.openshift.io/set-forwarded-headers, spec.httpHeaders.forwardedHeaderPolicy, spec.httpHeaders.actions in IngressController                                    | spec.httpHeaders.actions                                                                                                         |
+| ABD	                   | haproxy.router.openshift.io/set-forwarded-headers, spec.httpHeaders.forwardedHeaderPolicy, spec.httpHeaders.actions in Route                                                | spec.httpHeaders.actions                                                                                                         |
+| ACD                    | haproxy.router.openshift.io/set-forwarded-headers,spec.httpHeaders.actions in IngressController,spec.httpHeaders.actions in Route                                           | spec.httpHeaders.actions in Route.                                                                                               |
+| BCD                    | 	spec.httpHeaders.forwardedHeaderPolicy, spec.httpHeaders.actions in IngressController, spec.httpHeaders.actions in Route                                                   | spec.httpHeaders.actions in Route.                                                                                               |
+| AB                     | haproxy.router.openshift.io/set-forwarded-headers, spec.httpHeaders.forwardedHeaderPolicy                                                                                   | haproxy.router.openshift.io/set-forwarded-headers will set the policy for handling X-Forwarded-For headers.                      |
+| AC                     | haproxy.router.openshift.io/set-forwarded-headers, spec.httpHeaders.actions in IngressController                                                                            | spec.httpHeaders.actions will set X-Forwarded-For.                                                                               |
```

Comment:
it doesn't make sense that if you set

> haproxy.router.openshift.io/set-forwarded-headers, spec.httpHeaders.forwardedHeaderPolicy, 
> spec.httpHeaders.actions in IngressController
> 

the value is taken from `spec.httpHeaders.actions`, but if you set only
```
spec.httpHeaders.forwardedHeaderPolicy, 
spec.httpHeaders.actions in IngressController
```
the value is sometimes affected by the the policy that is used.  Why does it matter for one case and not the other?  

Same question when using 
```
spec.httpHeaders.forwardedHeaderPolicy, 
spec.httpHeaders.actions in Route
```

Reply:
@miheer The chart was an idea for expressing the override behavior to users, because  it still isn't clear.  If you have a clear way of expressing the override behavior, let's get rid of the chart, replace it with a clear explanation, and perhaps then we can close this EP.